.github/workflows/fitbot.yml

```
@@ -10,8 +10,8 @@ on:
 env:
   N3FIT_MAXNREP: 20 # total number of replicas to fit
   POSTFIT_NREP: 16 # requested replicas for postfit
-  REFERENCE_SET: NNBOT-c0f99b7b3-2024-02-28 # reference set for exact results
-  STABLE_REFERENCE_SET: NNBOT-c0f99b7b3-2024-02-28 # reference set for last tag
+  REFERENCE_SET: NNBOT-3e7c84220-2024-02-28 # reference set for exact results
+  STABLE_REFERENCE_SET: NNBOT-3e7c84220-2024-02-28 # reference set for last tag
```

```suggestion
  STABLE_REFERENCE_SET: NNBOT-c0f99b7b3-2024-02-28 # reference set for last tag
```

For the tag we should keep the previous one!

---

Ah right, I thought you wanted to tag after this PR